scripts/verify-docs.py

```diff
@@ -46,6 +46,12 @@
 test_config_filename = 'config.yaml'
 exec_script_filename = 'exec.sh'
 
+required_fields = ['description', 'tags', 'emails', 'license']
```

Comment:
Those fields are required by CI to generate the https://catalog.kpt.dev/ content. I think changing this could break the website 😞

Reply:
I did a quick look at the verify-docs.py. An easier solution to have our pilot functions moving fast forward and not break the website could be to add the set-gcp-resource-ids to this white list
https://github.com/GoogleContainerTools/kpt-functions-catalog/blob/56c6212166a38ed10fc7d24e69192c5d28dedf9b/scripts/verify-docs.py#L39

In the long term, I'd like to have the catalog function infra simplified and give authors more freedom to make changes. it is currently designed for a mature function eco-system and set some hard requirements for adopters to give good documentations and have the website up to date. I think it is the right way for user guide in the long run, but may be an overkill for our pivot function developing and exploring cycle.

Reply:
Here's the whitelist https://github.com/GoogleContainerTools/kpt-functions-catalog/blob/56c6212166a38ed10fc7d24e69192c5d28dedf9b/scripts/verify-docs.py#L39